gateway/vendor.conf

```diff
@@ -1,3 +1,4 @@
+github.com/docker/cli/opts 08097edc7849e1543d723a3c382536909b58aa4f
```

Comment:
What change needed the new vendored component?

Reply:
This was for processing secrets from strings, this allows for using the same style of arguments that the cli accepts.  For example 

```
source=mysql_password,target=wp_db_password,mode=0400
```
like they document at https://docs.docker.com/engine/swarm/secrets/#advanced-example-use-secrets-with-a-wordpress-service